integration_tests/run_pyspark_from_build.sh

```diff
@@ -80,7 +128,7 @@ else
         # Make sure we have Parquet version >= 1.12 in the dependency
         LOWEST_PARQUET_JAR=$(echo -e "$MIN_PARQUET_JAR\n$PARQUET_HADOOP_TESTS" | sort -V | head -1)
         export INCLUDE_PARQUET_HADOOP_TEST_JAR=$([[ "$LOWEST_PARQUET_JAR" == "$MIN_PARQUET_JAR" ]] && echo true || echo false)
-        PLUGIN_JARS=$(echo "$TARGET_DIR"/../../dist/target/rapids-4-spark_*.jar)
+        PLUGIN_JAR=${PLUGIN_JAR:-$(echo "$TARGET_DIR"/../../dist/target/rapids-4-spark_*.jar)}
```

Comment:
This is actually part of the bigger classpath mess we have. Our code should check that if we use spark.jars or spark.packages to specify the plugin artifact it should not include the one from TARGET_DIR

Reply:
Is there an issue created to overhaul this "classpath mess"? I'd rather not try to handle that mess in this PR.

Reply:
We can defer the class path cleanup to a dedicated issue. But the previous ask to update the "Running with Alternate Paths" section in the integration tests README.md is essential for this PR

Reply:
I already made the changes to the README file. PTAL. Let me know if you feel we want to add more to it.